Gruntfile.coffee

```diff
@@ -30,28 +31,61 @@ module.exports = ->
       fbp_test:
         command: 'fbp-test --colors'
 
+    # Building the website
+    jekyll:
+      options:
+        src: 'site/'
+        dest: 'dist/'
+        bundleExec: true
+      dist:
+        options:
+          dest: 'dist/'
+      serve:
+        options:
+          dest: 'dist/'
+          serve: true
+          watch: true
+          host: process.env.HOSTNAME or 'localhost'
+          port: process.env.PORT or 4000
+    # Deploying the website
+    'gh-pages':
+      options:
+        base: 'dist/',
+        clone: 'gh-pages'
+        message: "Release #{pkg.name} #{process.env.TRAVIS_TAG}"
+        repo: 'https://' + process.env.GH_TOKEN + '@github.com/msgflo/msgflo.git'
+        user:
+          name: 'msgflo bot',
+          email: 'jononor+msgflobot@gmail.com'
+        silent: true
+      src: '**/*'
 
   # Grunt plugins used for building
+  @loadNpmTasks 'grunt-jekyll'
 
   # Grunt plugins used for testing
   @loadNpmTasks 'grunt-mocha-test'
   @loadNpmTasks 'grunt-coffeelint'
   @loadNpmTasks 'grunt-shell-spawn'
 
+  # For deploying
+  @loadNpmTasks 'grunt-gh-pages'
+
   # Our local tasks
   @registerTask 'fbp-test', [
     'shell:msgflo'
     'shell:fbp_test'
     'shell:msgflo:kill'
   ]
 
-  @registerTask 'build', 'Build the chosen target platform', (target = 'all') =>
-    # nothing
-
   @registerTask 'test', 'Build and run automated tests', (target = 'all') =>
     @task.run 'coffeelint'
-    @task.run 'build'
     @task.run 'mochaTest'
 #    @task.run 'fbp-test'
 
   @registerTask 'default', ['test']
+
+  @registerTask 'site', [
```

Comment:
This does deploy also. Should have a name which makes this clear, like `deploysite` or `site:deploy`. Otherwise a dev may trigger it locally by accident and overwrite production site, when they just try to build something...